docs/integrations/app-integrations/microsoft-teams.mdx

```diff
@@ -35,60 +35,75 @@ Required connection parameters include the following:
+### How to set up Microsoft Teams app
 
 Follow the instructions given below to set up the Microsoft Teams app that will act as the chatbot:
 
-  1. Follow [this link](https://dev.botframework.com/bots/new) to the Microsoft Bot Framework portal and sign in with your Microsoft account.
-  2. Fill out the *Display name*, *Bot handle*, and, optionally, the *Long description*, but leave the *Messaging endpoint* field empty for now.
-  3. Set the *App type* to be `Multi Tenant` and click on *Create Microsoft App ID and password*. This will open a new tab with the Azure portal.
-  4. Click on *New registration* and fill out the *Name* and select the `Accounts in any organizational directory (Any Azure AD directory - Multitenant)` option under *Supported account types*, and click on *Register*. Record the *Application (client) ID* for later use.
-  5. Click on *Certificates & secrets* under *Manage*.
-  6. Click on *New client secret* and fill out the *Description* and select an appropriate *Expires* period, and click on *Add*.
-  7. Copy the client secret and save it in a secure location.
+<Steps>
+  <Step title="Sign in to Microsoft and fill in the bot data">
+    - Follow [this link](https://dev.botframework.com/bots/new) to the Microsoft Bot Framework portal and sign in with your Microsoft account.
+    - Fill out the *Display name*, *Bot handle*, and, optionally, the *Long description*, but leave the *Messaging endpoint* field empty for now.
+    - Set the *App type* to be `Multi Tenant` and click on *Create Microsoft App ID and password*. This will open a new tab with the Azure portal.
+  </Step>
+  <Step title="Set up the Azure portal">
+    - Click on *New registration* and fill out the *Name* and select the `Accounts in any organizational directory (Any Azure AD directory - Multitenant)` option under *Supported account types*, and click on *Register*. **Save the *Application (client) ID* for later use.**
+    - Click on *Certificates & secrets* under *Manage*.
+    - Click on *New client secret* and fill out the *Description* and select an appropriate *Expires* period, and click on *Add*.
+    - Copy and **save the client secret in a secure location.**
+    <Tip>
+    If you already have an existing app registration, you can use it instead of creating a new one and skip the above steps.
+    </Tip>
+  </Step>
+  <Step title="Set up the MindsDB editor">
+    - Open the MindsDB editor and create a connection to Microsoft Teams using the client ID and client secret obtained in the previous steps. Use the `CREATE DATABASE` statement as show above.
+    - Using this connection, create a chatbot with the [`CREATE CHATBOT`](/agents/chatbot) syntax, as shown in the Usage section below.
+    - Run the `SHOW CHATBOTS` command and record the `webhook_token` of the chatbot you created.
+  </Step>
+  <Step title="Set up the Microsoft Bot Framework portal">
```

Comment:
It might be a little confusing to say this because we are coming back to the Bot Framework portal after working on it earlier.

Reply:
CHanged it to `Complete the bot setup in the Microsoft Bot Framework portal`